src/controller/Controller.spec.js

```diff
@@ -106,4 +106,23 @@ describe('controller', () => {
       done();
     }, 10);
   });
+
+  it('should map "left" to "leftarrow" on IE ', function() {
+
+    if (!isIE11) {
```

Comment:
I think a better way to approach this test is to _make_ an event that mimics IE11's behavior rather than skip the test when not in IE. If we skip the test, we'll be less likely to catch it if it begins to break.